runtime/near-vm-errors/src/lib.rs

```diff
@@ -34,12 +32,8 @@ pub enum FunctionCallError {
     WasmUnknownError,
     HostError(HostError),
     EvmError(EvmError),
-    /// An error message when wasmer 1.0 returns a wasmer::RuntimeError
-    WasmerRuntimeError(String),
-    /// A trap in Wasmer 1.0, not same as WasmTrap above, String is a machine readable form like "stk_ovf"
-    /// String is used instead of wasmer internal enum is because of BorshSerializable.
-    /// It can be convert back by wasmer_vm::TrapCode::from_str
-    Wasmer1Trap(String),
+    /// Non-deterministic error.
+    Nondeterministic(String),
```

Comment:
Is this error deterministic, i.e, is it guaranteed that a nondeterministic error will be wrapped in this variant?

Reply:
Yes, now it is.